test/controllers/IllPersonController.test.ts

```diff
@@ -0,0 +1,63 @@
+import { expect } from 'chai';
+import request from 'supertest';
+import { createApp } from '../ApplicationFactory';
+
+import { Appeal } from 'app/models/Appeal';
+import { ApplicationData } from 'app/models/ApplicationData';
+import { ILL_PERSON_PAGE_URI } from 'app/utils/Paths';
+
+describe('IllPersonController', () => {
+
+    const applicationData: Partial<ApplicationData> = {
+        appeal: {
+            penaltyIdentifier: {
+                companyNumber: 'NI000000'
+            }
+        } as Appeal,
+        navigation: { permissions: [ILL_PERSON_PAGE_URI] }
+    };
+
+    it('should show radio buttons for available Ill Person options on GET', async () => {
```

Comment:
something we have done in a few of the tests which im quite fond of is separate GETS and POST tests.

What I mean exactly is: 

- Within the main describe, you can add one for GET and another one for POST. 
```
describe('IllPersonController', () => {
    describe('GET request', () => {}
    describe('POST request', () => {}
}
```
You can then remove the "GET" and "POST" from the names of the tests. Keeps it organized and easier to read.

Reply:
Ah yes, I've seen this in some cases but not others, the inconsistency is upsetting. I prefer this so I'll switch to it. Might be nice to refactor all of the controller tests to use this pattern in future.